docs/src/componentDocs/ListItemTag/examples/ListItemTagUsingInfoListItem.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+import {ListItemTagUsingInfoListItemExample} from './ListItemTagUsingInfoListItemExample';
+
+const codeSnippet = `<InfoListItem
+    icon={{name: 'battery-charging-full'}}
+    iconColor="#528AAE"
+    title="Info List Item"
+    subtitle="with a ListItemTag component to the right"
+    rightComponent={
+      <View style={{flexDirection: 'row'}}>
+          <ListItemTag
+              label="Build Passing"
+              backgroundColor={Colors.green[300]}
```

Comment:
Can we use Hex code instead colors.green[300]? RN components use MD3 colors and the react-native docs app is in react which uses MD2 colors.